src/ducks/settings/AccountsListSettings.jsx

```diff
@@ -106,6 +58,28 @@ const AccountsListSettings = ({
     connectionId: getConnectionIdFromAccount(accounts[0])
   }))
 
+  const slugInMaintenance = useMemo(
+    () => appsInMaintenance.map(appsInMaintenance => appsInMaintenance.slug),
+    [appsInMaintenance]
+  )
+
+  useEffect(() => {
+    const fetchTriggers = async () => {
+      const result = await client.query(
+        cronKonnectorTriggersConn.query(client),
+        {
+          as: cronKonnectorTriggersConn.as
+        }
+      )
+      const errors = result.data
+        .filter(trigger => get(trigger, 'current_state.status') === 'errored')
+        .map(trigger => get(trigger, 'message.konnector'))
+      setSlugInError(errors)
+    }
+    fetchTriggers()
+    // eslint-disable-next-line react-hooks/exhaustive-deps
+  }, [])
```

Comment:
Why not using `useQuery`? 

```jsx
const results = useQuery(cronKonnectorTriggersConn.query, cronKonnectorTriggersConn.as)
if(results.data){
 const errors = result.data
        .filter(trigger => get(trigger, 'current_state.status') === 'errored')
        .map(trigger => get(trigger, 'message.konnector'))
      setSlugInError(errors)
}
```

It brings you an API compatible with others queries by returning a fetchStatus for instance. This way, you can wait for this query to be loaded before displaying something?

Without by default the `AccountListItem` will be first displayed without any warning or else, and then flicker to be displayed is maintenance or in error. Is it the wanted behavior?

Reply:
Hmmm with useQuery I have a `Uncaught Invariant Violation: Too many re-renders. React limits the number of renders to prevent an infinite loop.`.